src/api/taxonomyApi.ts

```diff
@@ -145,18 +145,32 @@ export async function fetchTopicResources(
   const subjectParam = subjectId ? `&subject=${subjectId}` : '';
 
   const response = await fetch(
-    `/taxonomy/v1/topics/${topicId}/resources?relevance=${relevance}&language=${context.language}${filterParam}${subjectParam}`,
+    `/taxonomy/v1/topics/${topicId}/resources?language=${context.language}${filterParam}${subjectParam}`,
     context,
   );
   const resources: GQLTaxonomyEntity[] = await resolveJson(response);
-
   resources.forEach(resource => {
     if (subjectId) {
       const primaryPath = findPrimaryPath(resource.paths, subjectId);
       const path = primaryPath ? primaryPath : resource.path;
       resource.path = path;
     }
   });
+
+  const suplResources = resources.filter(resource =>
+    resource.filters?.find(
+      filter => filter.relevanceId === 'urn:relevance:supplementary',
+    ),
+  );
+  const coreResources = resources.filter(
+    resource => !suplResources.includes(resource),
+  );
+  if (relevance === 'urn:relevance:core') {
```

Comment:
Så dette er basicaly en flytting av filtere fra backend itl frontend?

Reply:
Ja. Dette er nødvendig fordi filter fjernes for tverrfaglige tema, og då har vi ikkje lenger relevans å filtere på i backend. Relevance skal flyttes fra filter til koblingsobjektet mellom emne og ressurs, og då kan vi endre til å sende inn relevance dit men det er ikkje klart i taksonomi endå. For alle ressurser som har filter og som har relevance så vil dette oppføre seg som tidligere.